Requirements/qiskit.txt

```diff
@@ -1,5 +1,6 @@
 numpy
 pandas
+ipykernel
```

Comment:
Added ipykernel to the qiskit.txt installation, as creating a new file with Jupyter in a codespace would not allow any python to be ran without it. 
It would always prompt to "install ipykernel" so I figured this may be a good addition as it fixed the issue entirely for me without need of inputs.